README.md

```diff
@@ -1,2 +1,31 @@
 # AutomatedTesting
-Starter Code for Automated Testing into in Ruby
+
+## Learning Goals
+This exercise is designed to enable you to:
+- Learn Minitest spec-style syntax
+- Learn to design spec-style tests to adequately ensure a class performs as expected.
+
+## Overview
```

Comment:
Since this is a very early project with testing, it would be good to have a little explanation of the files and the file structure.